.nvmrc

```diff
@@ -0,0 +1 @@
+16.14
```

Comment:
There's a `.nvmrc` file in `frontend` already 😉.

Reply:
there's a package.json in the frontend folder, and another one at the root.